blocks/library/paragraph/index.js

```diff
@@ -198,7 +205,7 @@ registerBlockType( 'core/paragraph', {
 		const styles = {
 			backgroundColor: backgroundColor,
 			color: textColor,
-			fontSize: fontSize,
+			fontSize: fontSize ? fontSize + 'em' : undefined,
```

Comment:
What happens if the theme sets a font size on the paragraph in a stylesheet? Then it would base the font size on the parent node, right? https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#Ems

Reply:
Yeah, that's why I thought that `em` was the right choice. The theme should be setting the base font size, and the font size slider is there to make it bigger or smaller. Or am I misunderstanding the issue?

Reply:
Yes, `em` works in that way, but ideally it would be relative to the normal font size of that element, rather than the font size of the parent element. See https://github.com/WordPress/gutenberg/issues/2610#issuecomment-331666206. Many themes target the paragraph directly, e.g. `.content p { font-size: 16px }`. In this case, I think it would break?

Reply:
I'm not sure... from what I can see, it would be fine. Here's how we currently set the font size on paragraphs in the editor:

```
.editor-visual-editor p {
    font-family: "Noto Serif", serif;
    font-size: 16px;
    line-height: 1.8;
}
```

Applying a `font-size` in the paragraph's style attribute seems to accept 16px as the same as 1em, and it works as I expected. (I compared an unstyled paragraph to one with font-size set to 1em, they were identical) Perhaps I'm missing some detail though?

Reply:
I was missing a detail. It was coincidence it worked. Thanks for putting me on the right track here.